### PR TITLE
SLNS-73 Vehicle Stats

### PR DESCRIPTION
**Pull request by ItsCalebJones**

### CODE REVIEW

File: api/models.py [hunk 1]
```diff
@@ -844,8 +869,6 @@ class FirstStage(models.Model):
     launcher = models.ForeignKey(Launcher, related_name='firststage', on_delete=models.CASCADE)
     rocket = models.ForeignKey(Rocket, related_name='firststage', on_delete=models.CASCADE)
 
-    # Not a property but keeping around as a helper
-
     @property
     def previous_flight(self):
```
Comment (JRascagneres) [inline api/models.py]: Any reason this is added back as a property?

File: api/models.py [hunk 2]
```diff
@@ -1169,8 +1192,7 @@ def orbital_launch_attempt_count(self):
             return count
 
         if not self.mission.orbit or self.mission.orbit.name != "Sub-Orbital":
```
Comment (JRascagneres) [inline api/models.py]: Can remove "Sub-Orbital" if check as its not in the filter

File: api/models.py (hunk 1 above)
Comment (ItsCalebJones) [reply api/models.py]: Shouldnt cause any problems but no - not sure why it happened.

Comment (JRascagneres) [reply api/models.py]: Yeah no problems, just unnecessary

File: api/models.py (hunk 2 above)
Comment (JRascagneres) [reply api/models.py]: Misunderstanding

**Pull request merged**